packages/web-components/src/components/ic-toggle-button/ic-toggle-button.tsx

```diff
@@ -84,6 +88,11 @@ export class ToggleButton {
    */
   @Prop({ mutable: true, reflect: true }) toggleChecked: boolean = false;
 
+  /**
+   * The position of the tooltip in relation to the icon-only toggle button.
```

Comment:
You can display a tooltip on any variant of toggle buttons by using the `accessibleLabel` prop, so I think remove the `icon-only`?

```suggestion
   * The position of the tooltip in relation to the toggle button.
```

Reply:
Good spot, updated!